packages/babel-plugin/src/utils/ast.tsx

```diff
@@ -241,20 +343,24 @@ export const getInterpolation = (expression: t.Expression, meta: Metadata): t.Ex
 
   if (t.isIdentifier(expression)) {
     const binding = meta.parentPath.scope.getBinding(expression.name);
-    if (binding && binding.constant && t.isVariableDeclarator(binding.path.node)) {
-      value = binding.path.node.init;
+    const resolvedBinding = resolveBindingNode(binding, meta);
+
+    if (resolvedBinding && resolvedBinding.constant) {
+      // We recursively call get interpolation until it not longer returns an identifier or member expression
+      value = getInterpolation(resolvedBinding.node as t.Expression, resolvedBinding.meta);
     }
   } else if (t.isMemberExpression(expression)) {
     const { accessPath, bindingIdentifier } = getMemberExpressionMeta(expression);
     const binding = meta.parentPath.scope.getBinding(bindingIdentifier.name);
-
-    if (
-      binding &&
-      binding.constant &&
-      t.isVariableDeclarator(binding.path.node) &&
-      t.isObjectExpression(binding.path.node.init)
-    ) {
-      value = getValueFromObjectExpression(binding.path.node.init, accessPath);
+    const resolvedBinding = resolveBindingNode(binding, meta);
+
+    if (resolvedBinding && resolvedBinding.constant && t.isObjectExpression(resolvedBinding.node)) {
+      const objectValue = getValueFromObjectExpression(
+        resolvedBinding.node,
+        accessPath
+      ) as t.Expression;
+      // We recursively call get interpolation until it not longer returns an identifier or member expression
+      value = getInterpolation(objectValue, resolvedBinding.meta);
     }
   }
 
```

Comment:
add a comment on line 371? talking about potential perf pitfalls of the current recursive solution and to watch out for later - we may need to optimize this code path later.